mobile-app/app/screens/AppNavigator/screens/Loans/screens/AddOrRemoveCollateralScreen.tsx

```diff
@@ -0,0 +1,803 @@
+                    }
+                  )}
+            </ThemedTextV2>
+          )}
+
+        {/* Conversion required warning */}
+        {isConversionRequired && (
+          <ThemedTextV2
+            light={tailwind("text-orange-v2")}
+            dark={tailwind("text-orange-v2")}
+            style={tailwind("text-xs pt-2 mx-5 font-normal-v2")}
+            testID="conversion_required_warning"
+          >
+            {translate(
+              "screens/AddOrRemoveCollateralScreen",
+              "A small amount of UTXO is reserved for fees"
+            )}
+          </ThemedTextV2>
+        )}
+
+        {/* Insufficient balance error */}
+        {formState.errors.collateralAmount?.type === "hasSufficientFunds" && (
+          <ThemedTextV2
+            light={tailwind("text-red-v2")}
+            dark={tailwind("text-red-v2")}
+            style={tailwind("text-xs pt-2 mx-5 font-normal-v2")}
+            testID="add_remove_collateral_insufficient_balance"
+          >
+            {translate(
+              "screens/AddOrRemoveCollateralScreen",
+              isAdd
+                ? "Insufficient Balance"
+                : "Amount entered is higher than collateral"
+            )}
+          </ThemedTextV2>
+        )}
+
+        {/* Min vault share required error */}
+        {displayMinVaultShareRequiredError && formState.isValid && (
+          <ThemedTextV2
+            light={tailwind("text-red-v2")}
+            dark={tailwind("text-red-v2")}
+            style={tailwind("text-xs pt-2 mx-5 font-normal-v2")}
+            testID="vault_min_share_warning"
+          >
+            {translate(
+              "screens/BorrowLoanTokenScreen",
+              "Vault needs at least 50% DFI share"
+            )}
+          </ThemedTextV2>
+        )}
+      </View>
+
+      <ThemedViewV2
+        light={tailwind("border-mono-light-v2-300")}
+        dark={tailwind("border-mono-dark-v2-300")}
+        style={tailwind("p-5 mt-6 border-0.5 rounded-lg-v2")}
+      >
+        <ThemedViewV2
+          light={tailwind("border-mono-light-v2-300")}
+          dark={tailwind("border-mono-dark-v2-300")}
+        >
+          <TextRowV2
+            containerStyle={{
+              style: tailwind("flex-row items-start w-full"),
+              light: tailwind("bg-transparent"),
+              dark: tailwind("bg-transparent"),
+            }}
+            lhs={{
+              value: translate(
+                "screens/AddOrRemoveCollateralScreen",
+                "Vault ID"
+              ),
+              testID: "add_remove_collateral_vault_id_label",
+              themedProps: lhsThemedProps,
+            }}
+            rhs={{
+              value: vault.vaultId,
+              testID: `add_remove_collateral_vault_id`,
+              numberOfLines: 1,
+              ellipsizeMode: "middle",
+              themedProps: rhsThemedProps,
+            }}
+          />
+          <NumberRowV2
+            info={{
```

Comment:
This was added in the existing `NumberRowV2` component, probably because of android:
![image](https://user-images.githubusercontent.com/17063281/195773348-bb4a2ed2-8472-46ea-ac5b-bc7be3b8e396.png)

Reply:
Can we conditionally add this for specific platform?

Reply:
Sure we can. I just updated, only added for android.